### ISSUE TITLE
align stuff better

### ISSUE THREAD

**Issue by andrewrk** [andrewrk/poop#4]
example:
```
  measurement      mean ± σ               min … max                  delta
  wall_time        1.254s ± 35.23ms       1.231s … 1.294s            ⚡-3.0%
  peak_rss         169M ± 97K             169M … 169M                  +0.4%
  cpu_cycles       4639444330 ± 42490202  4597639737 … 4682588462    ⚡-1.2%
  instructions     5998472007 ± 1915499   5996326492 … 6000010285      -0.1%
  cache_references 264926528 ± 1161752    263873524 … 266172790        -0.5%
  cache_misses     26841904 ± 359513      26436439 … 27121761        ⚡-2.1%
  branch_misses    35020581 ± 56286       34959963 … 35071194          -0.8%
```

it would be nice if the +/- and the ... lined up with each other

**Comment by andrewrk** [comment]
Fix reverted in b8b4297a067033feab1faf18c23615c7d4efd06f

**Comment by andrewrk** [comment]
Fixed by #30
